index.js

```diff
@@ -2,7 +2,6 @@ var pjson = require('./package.json')
 
 exports.name = 'agol'
 exports.hosts = true
-exports.pattern = '/:id/:item'
```

Comment:
correct.
Two reasons:
1. pattern is not well understood or documented
2. it prevents me from using the shared controller method to pass through information